### ISSUE TITLE
Flags to opt in to  .NET Framework 4.7.1, 4.7.2, 4.8, and the August 2020 cumulative update for .NET Framework 4.8 accessibility changes are wrong

### ISSUE THREAD

**Issue by MattHyman** [dotnet/docs#29758]
Flags to opt in to  .NET Framework 4.7.1, 4.7.2, 4.8, and the August 2020 cumulative update for .NET Framework 4.8 accessibility changes are wrong; quotations should be around the value and semicolons separating each flag instead of lines.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 642b7f23-cc05-b8cb-5ec1-862fa6e62b8c
* Version Independent ID: 0d7236b7-8c64-7eb0-69c8-101c738f275a
* Content: [What's new in accessibility in .NET Framework](https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility)
* Content Source: [docs/framework/whats-new/whats-new-in-accessibility.md](https://github.com/dotnet/docs/blob/main/docs/framework/whats-new/whats-new-in-accessibility.md)
* Product: **dotnet-framework**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

**Comment by gewarren** [comment]
Thanks @MattHyman! I've fixed this in the linked PR.
